.env

```diff
@@ -1,24 +1,25 @@
 NODE_ENV='production'
-BASE_URL=null
```

Comment:
If you provide any value other than an empty string (`''`), it parses as a string when reading the `process.env` variable. Null and falsey values have been changed to an empty string.